### PR TITLE
Merge code from oap common

### PR DESCRIPTION
**Pull request by yeyuqiang**

### CODE REVIEW

**Comment by yeyuqiang** [comment]
@winningsix @ZJie1

**Comment by winningsix** [comment]
LGTM +1

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.intel</groupId>
+    <artifactId>elastic-data-fusion</artifactId>
+    <version>0.8.0</version>
```
Comment (jikunshang) [inline pom.xml:20]: 0.1.0 or something else? 0.8.0 here is wierd.

Comment (winningsix) [reply pom.xml:20]: Good catch! I guess it's aligned with OAP version. I would suggest to start with 0.0.1.

**Comment by jikunshang** [comment]
LGTM

**Pull request merged**